repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -473,6 +473,35 @@ const createProductDetails = async (
         });
     }
 
+    if ('pricingByDistance' in product && product.pricingByDistance) {
+        const pricePerDistance = product.pricingByDistance;
```

Comment:
```suggestion
        const { pricingPerDistance } = product;
```
Calling the variable pricePerDistance makes it sound like it is just one price.